From 8ce1c0b75c9bf014d5117c87a67488fc4d58b2a6 Mon Sep 17 00:00:00 2001
From: Derek Higgins <derekh@redhat.com>
Date: Tue, 15 Jul 2014 22:45:07 +0000
Subject: [PATCH 1/2] Remove references to xstatic

This isn't available yet and apparently is not yet required.
---
 openstack_dashboard/settings.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/openstack_dashboard/settings.py b/openstack_dashboard/settings.py
index e99a9af..d36e249 100644
--- a/openstack_dashboard/settings.py
+++ b/openstack_dashboard/settings.py
@@ -22,8 +22,6 @@ import sys
 import warnings
 
 from django.utils.translation import ugettext_lazy as _
-import xstatic.main
-import xstatic.pkg.jquery
 
 from openstack_dashboard import exceptions
 
@@ -136,7 +134,7 @@ STATICFILES_FINDERS = (
 )
 
 STATICFILES_DIRS = (
-    ('horizon/lib/jquery', xstatic.main.XStatic(xstatic.pkg.jquery).base_dir),
+    ('horizon/lib/jquery',),
 )
 
 COMPRESS_PRECOMPILERS = (
-- 
1.9.3

